discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -237,6 +233,15 @@ def wait_for_non_bootstrap_masters_to_reach_configuring_state_during_install(sel
             nodes_count=env_variables['num_masters']-1
         )
 
+    def wait_for_non_bootstrap_masters_to_reach_joined_state_during_install(self):
```

Comment:
Why not making this method more generic by calling it `wait_for_hosts_to_reach_joined`, and accept `nodes_count` as a parameter?